packages/router/src/directives/router_outlet.ts

```diff
@@ -349,3 +369,56 @@ class OutletInjector implements Injector {
     return this.parent.get(token, notFoundValue);
   }
 }
+
+export const INPUT_BINDER = new InjectionToken<RoutedComponentInputBinder|null>('');
```

Comment:
```suggestion
export const INPUT_BINDER = new InjectionToken<RoutedComponentInputBinder>('');
```

(`inject` with `optional: true` adds `null` to the type already, it's not needed here)